src/Core/Identity/WebAuthnTokenProvider.cs

```diff
@@ -77,7 +77,7 @@ public async Task<string> GenerateAsync(string purpose, UserManager<User> manage
             var providers = user.GetTwoFactorProviders();
             providers[TwoFactorProviderType.WebAuthn] = provider;
             user.SetTwoFactorProviders(providers);
-            await userService.UpdateTwoFactorProviderAsync(user, TwoFactorProviderType.WebAuthn);
+            await userService.UpdateTwoFactorProviderAsync(user, TwoFactorProviderType.WebAuthn, true, false);
```

Comment:
I think the following pattern is easier to read.
```suggestion
            await userService.UpdateTwoFactorProviderAsync(user, TwoFactorProviderType.WebAuthn, logEvent: true);
```

Reply:
Neeeeeat. Updated!